Resources/Storage/StorageBase.py

```diff
@@ -415,3 +417,15 @@ def _isInputURL( self, url ):
       return S_OK( True )
 
     return S_OK( urlDict['Protocol'] == self.protocolParameters['Protocol'] )
+
+  #############################################################
+  #
+  # These are the methods for getting information about the Storage element:
+  #
+
+  def getOccupancy(self, *parms, **kws):
+    """ Get the StorageElement occupancy info.
+        :returns: S_OK/S_ERROR dictionary
+    """
+    #FIXME: put an implementation that just gets a file and check its content
+    return S_ERROR("Storage.occupancy: implement me!")
```

Comment:
`StorageBase.getOccupancy`

Reply:
Well all the other return codes in this module contains `Storage.method:`so I just left it like the others.